test/performance/io/format_fasta_benchmark.cpp

```diff
@@ -42,6 +44,12 @@ void write3(benchmark::State & state)
     {
         format.write(ostream, options, seq, id, std::ignore);
     }
+
+    ostream = std::ostringstream{};
```

Comment:
why do you need to write to a flushed sotream again? can't you just use `ostream.size()` because we have already written to it?